src/table_builder.cc

```diff
@@ -146,7 +146,8 @@ void TitanTableBuilder::AddBlob(const ParsedInternalKey& ikey,
 
   // Init blob_builder_ first
   if (!blob_builder_) {
-    status_ = blob_manager_->NewFile(&blob_handle_);
+    // Set the Flush's blob file with a high_io pri in ratelimiter
+    status_ = blob_manager_->NewFile(&blob_handle_, Env::IOPriority::IO_HIGH);
```

Comment:
the blob file may be generated during compaction, for compaction seems we should set low pri? you can check `target_level_` to decide if it's a flush or compaction